update to glibc 2.32

t-lo opened the pull request:
# This PR updates sys-libs/glibc to 2.32.
* import glibc-2.32 Gentoo upstream recipe
* back-port glibc-2.32 recipe to EAPI6 because our outdated portage doesn't handle BDEPEND well
* include a patch to fix a test in `net-fs/samba` 4.11.13 to fix build brekage (see [upstream discussion](https://forums.gentoo.org/viewtopic-t-1118902.html?sid=e1809c2d407bafda1df63d9115bcaaef))  
* add "crypt" to forced USE flags to make the SDK glibc build include libcrypt
* add a patch for libselinux to use glibc's `gettid()` which is shipped with glibc-2.30 and above
* add a patch for fero-client's grpcio-sys to use glibc's `gettid()` which is shipped with glibc-2.30 and above
* add `dev-util/cmake` dependency to fero-client which was missing previously

# Testing done:
- SDK build, toolchain build, packages build and images build all succeeded
- qemu and qemu_uefi tests succeed
- Diff of OS image contents and dev container contents to Nov-25 nightly do not show anything suspicious (see below).

marga-kinvolk commented on coreos-devel/fero-client/fero-client-9999.ebuild:
```diff
@@ -23,14 +23,25 @@ SLOT="0"
 }
 
+src_prepare() {
+	# grpc-sys-0.2.3 crate build breaks with glibc-2.30 and above (see patch)
```
Is this "crate" here a typo for create? Or is it a noun and I'm missing some context?

Regardless of that, I'm a bit confused as to why this is patching grpc-sys in the fero-client package. This seems wrong... Why not patch grpc-sys?

pothos replied:
Crates are a Rust term for libraries on the official library hub. One could try to patch it on the Cargo level but at that point it's even easier to update the dependency since we would already touch the fero repo.

t-lo replied:
"crate" is a [rust crate](https://crates.io/), i.e. a package. We don't include grpc-sys anywhere, it's downloaded from the Rust crates repo during fero-client's build. I'll elaborate the comment in src_prepare.

marga-kinvolk commented on coreos-devel/fero-client/files/0001-gettid-glibc-2.30.patch:
```diff
@@ -0,0 +1,20 @@
+--- gentoo/grpcio-sys-0.2.3/grpc/src/core/lib/support/log_linux.c   2020-11-23 18:29:18.053038547 +0000
```
Could you add the explanation about why this patch is needed to the patch itself instead of having it in a different file?

marga-kinvolk commented on net-fs/samba/files/samba-4.11-fix-glibc-2.32-function-collisions.patch:
```diff
@@ -0,0 +1,76 @@
+Quick-and-dirty fix for Samba build issue with glibc-2.32,
```
Quick-and-dirty conveys zero information. I'd rather this explained the actual issue. Something like "Rename Samba functions that collide with new functions shipped in glibc-2.32" or similar.

marga-kinvolk commented on sys-libs/libselinux/libselinux-2.4.ebuild:
```diff
@@ -39,6 +39,7 @@ src_prepare() {
 	epatch "${FILESDIR}/0005-use-ruby-include-with-rubylibver.patch"
 	epatch "${FILESDIR}/0006-build-related-fixes-bug-500674.patch"
     epatch "${FILESDIR}/0007-fix-setexeccon-on-exec.patch"
+    epatch "${FILESDIR}/0008-do-not-define-gettid-for-glibc-2.30-and-above.patch"
```
There's some whitespace weirdness going on in this line and the line above.

t-lo commented:
Updated:
- Addressed all comments above
- added a new change to [glibc-2.32-r2's pkg_pretend](https://github.com/kinvolk/coreos-overlay/blob/f14c40cf5c884f119d7a2b8d4a8538ed45d28c4d/sys-libs/glibc/glibc-2.32-r2.ebuild#L730) to un-break the OS image build.

t-lo commented:
Testing done:
- SDK build, toolchain build, packages build and images build all succeeded

t-lo commented:
# Production image diff (to Nov-25 nightly)
```
$ awk '{ s = ""; for (i = 8; i <= NF; i++) s = s $i " "; print s }' flatcar_production_image_contents.txt | sort >prod-image-cont.txt
$ awk '{ s = ""; for (i = 8; i <= NF; i++) s = s $i " "; print s }' flatcar_production_image_contents-glibc-2.32.txt | sort >prod-image-cont-glibc.txt
$ diff -ur prod-image-cont.txt prod-image-cont-glibc.txt
--- prod-image-cont.txt	2020-11-25 17:51:44.267558187 +0100
+++ prod-image-cont-glibc.txt	2020-11-25 17:18:06.258475796 +0100
@@ -1810,7 +1810,7 @@
 ./usr/lib64/girepository-1.0/Polkit-1.0.typelib 
 ./usr/lib64/girepository-1.0/PolkitAgent-1.0.typelib 
 ./usr/lib64/glib-2.0 
-./usr/lib64/glibc-2.29 
+./usr/lib64/glibc-2.32 
 ./usr/lib64/kernel 
 ./usr/lib64/kernel/install.d 
 ./usr/lib64/kernel/install.d/00-entry-directory.install 
@@ -1830,11 +1830,11 @@
 ./usr/lib64/krb5/plugins/preauth/test.so 
 ./usr/lib64/krb5/plugins/tls 
 ./usr/lib64/krb5/plugins/tls/k5tls.so 
-./usr/lib64/ld-2.29.so 
+./usr/lib64/ld-2.32.so 
 ./usr/lib64/ldb 
 ./usr/lib64/ldb/libldb-cmdline.so 
 ./usr/lib64/ldb/libreplace-ldb.so 
-./usr/lib64/ld-linux-x86-64.so.2 -> ld-2.29.so 
+./usr/lib64/ld-linux-x86-64.so.2 -> ld-2.32.so 
 ./usr/lib64/ld.so.conf 
 ./usr/lib64/ld.so.conf.d 
 ./usr/lib64/libacl.so 
@@ -1843,8 +1843,8 @@
 ./usr/lib64/libaio.so 
 ./usr/lib64/libaio.so.1.0.1 
 ./usr/lib64/libaio.so.1 -> libaio.so.1.0.1 
-./usr/lib64/libanl-2.29.so 
-./usr/lib64/libanl.so.1 -> libanl-2.29.so 
+./usr/lib64/libanl-2.32.so 
+./usr/lib64/libanl.so.1 -> libanl-2.32.so 
 ./usr/lib64/libanl.so -> ../../lib64/libanl.so.1 
 ./usr/lib64/libapol.so.4.4 
 ./usr/lib64/libapol.so.4 -> libapol.so.4.4 
@@ -1884,8 +1884,8 @@
 ./usr/lib64/libblkid.so 
 ./usr/lib64/libblkid.so.1.1.0 
 ./usr/lib64/libblkid.so.1 -> libblkid.so.1.1.0 
-./usr/lib64/libBrokenLocale-2.29.so 
-./usr/lib64/libBrokenLocale.so.1 -> libBrokenLocale-2.29.so 
+./usr/lib64/libBrokenLocale-2.32.so 
+./usr/lib64/libBrokenLocale.so.1 -> libBrokenLocale-2.32.so 
 ./usr/lib64/libBrokenLocale.so -> ../../lib64/libBrokenLocale.so.1 
 ./usr/lib64/libbsd.so.0.10.0 
 ./usr/lib64/libbsd.so.0 -> libbsd.so.0.10.0 
@@ -1900,7 +1900,7 @@
 ./usr/lib64/libbz2.so.1.0.8 
 ./usr/lib64/libbz2.so.1.0 -> libbz2.so.1.0.8 
 ./usr/lib64/libbz2.so.1 -> libbz2.so.1.0.8 
-./usr/lib64/libc-2.29.so 
+./usr/lib64/libc-2.32.so 
 ./usr/lib64/libcap-ng.so.0.0.0 
 ./usr/lib64/libcap-ng.so.0 -> libcap-ng.so.0.0.0 
 ./usr/lib64/libcap-ng.so -> libcap-ng.so.0.0.0 
@@ -1919,16 +1919,16 @@
 ./usr/lib64/libcom_err.so 
 ./usr/lib64/libcom_err.so.2.1 
 ./usr/lib64/libcom_err.so.2 -> libcom_err.so.2.1 
-./usr/lib64/libcrypt-2.29.so 
+./usr/lib64/libcrypt-2.32.so 
 ./usr/lib64/libcrypto.so.1.1 
 ./usr/lib64/libcrypto.so -> libcrypto.so.1.1 
 ./usr/lib64/libcryptsetup.so.12.6.0 
 ./usr/lib64/libcryptsetup.so.12 -> libcryptsetup.so.12.6.0 
 ./usr/lib64/libcryptsetup.so -> libcryptsetup.so.12.6.0 
-./usr/lib64/libcrypt.so.1 -> libcrypt-2.29.so 
+./usr/lib64/libcrypt.so.1 -> libcrypt-2.32.so 
 ./usr/lib64/libcrypt.so -> ../../lib64/libcrypt.so.1 
 ./usr/lib64/libc.so 
-./usr/lib64/libc.so.6 -> libc-2.29.so 
+./usr/lib64/libc.so.6 -> libc-2.32.so 
 ./usr/lib64/libcurl.so.4.6.0 
 ./usr/lib64/libcurl.so.4 -> libcurl.so.4.6.0 
 ./usr/lib64/libcurl.so -> libcurl.so.4.6.0 
@@ -1961,8 +1961,8 @@
 ./usr/lib64/libdhash.so.1.0.2 
 ./usr/lib64/libdhash.so.1 -> libdhash.so.1.0.2 
 ./usr/lib64/libdhash.so -> libdhash.so.1.0.2 
-./usr/lib64/libdl-2.29.so 
-./usr/lib64/libdl.so.2 -> libdl-2.29.so 
+./usr/lib64/libdl-2.32.so 
+./usr/lib64/libdl.so.2 -> libdl-2.32.so 
 ./usr/lib64/libdl.so -> ../../lib64/libdl.so.2 
 ./usr/lib64/libdw-0.178.so 
 ./usr/lib64/libdw.so.1 -> libdw-0.178.so 
@@ -2185,7 +2185,7 @@
 ./usr/lib64/liblzo2.so.2.0.0 
 ./usr/lib64/liblzo2.so.2 -> liblzo2.so.2.0.0 
 ./usr/lib64/liblzo2.so -> liblzo2.so.2.0.0 
-./usr/lib64/libm-2.29.so 
+./usr/lib64/libm-2.32.so 
 ./usr/lib64/libmemusage.so 
 ./usr/lib64/libmenu.so.6.1 
 ./usr/lib64/libmenu.so.6 -> libmenu.so.6.1 
@@ -2210,11 +2210,11 @@
 ./usr/lib64/libmpathpersist.so.0 
 ./usr/lib64/libmpathpersist.so -> libmpathpersist.so.0 
 ./usr/lib64/libm.so 
-./usr/lib64/libm.so.6 -> libm-2.29.so 
+./usr/lib64/libm.so.6 -> libm-2.32.so 
 ./usr/lib64/libmultipath.so.0 
 ./usr/lib64/libmultipath.so -> libmultipath.so.0 
-./usr/lib64/libmvec-2.29.so 
-./usr/lib64/libmvec.so.1 -> libmvec-2.29.so 
+./usr/lib64/libmvec-2.32.so 
+./usr/lib64/libmvec.so.1 -> libmvec-2.32.so 
 ./usr/lib64/libmvec.so -> ../../lib64/libmvec.so.1 
 ./usr/lib64/libncurses.so 
 ./usr/lib64/libncurses.so.6.1 
@@ -2284,25 +2284,25 @@
 ./usr/lib64/libnpth.so.0.0.6 
 ./usr/lib64/libnpth.so.0 -> libnpth.so.0.0.6 
 ./usr/lib64/libnpth.so -> libnpth.so.0.0.6 
-./usr/lib64/libnsl-2.29.so 
-./usr/lib64/libnsl.so.1 -> libnsl-2.29.so 
+./usr/lib64/libnsl-2.32.so 
+./usr/lib64/libnsl.so.1 -> libnsl-2.32.so 
 ./usr/lib64/libnspr4.so 
 ./usr/lib64/libnss_cache_oslogin-20200910.00.so 
 ./usr/lib64/libnss_cache_oslogin.so.2 -> libnss_cache_oslogin-20200910.00.so 
-./usr/lib64/libnss_compat-2.29.so 
-./usr/lib64/libnss_compat.so.2 -> libnss_compat-2.29.so 
+./usr/lib64/libnss_compat-2.32.so 
+./usr/lib64/libnss_compat.so.2 -> libnss_compat-2.32.so 
 ./usr/lib64/libnss_compat.so -> ../../lib64/libnss_compat.so.2 
-./usr/lib64/libnss_db-2.29.so 
-./usr/lib64/libnss_db.so.2 -> libnss_db-2.29.so 
+./usr/lib64/libnss_db-2.32.so 
+./usr/lib64/libnss_db.so.2 -> libnss_db-2.32.so 
 ./usr/lib64/libnss_db.so -> ../../lib64/libnss_db.so.2 
-./usr/lib64/libnss_dns-2.29.so 
-./usr/lib64/libnss_dns.so.2 -> libnss_dns-2.29.so 
+./usr/lib64/libnss_dns-2.32.so 
+./usr/lib64/libnss_dns.so.2 -> libnss_dns-2.32.so 
 ./usr/lib64/libnss_dns.so -> ../../lib64/libnss_dns.so.2 
-./usr/lib64/libnss_files-2.29.so 
-./usr/lib64/libnss_files.so.2 -> libnss_files-2.29.so 
+./usr/lib64/libnss_files-2.32.so 
+./usr/lib64/libnss_files.so.2 -> libnss_files-2.32.so 
 ./usr/lib64/libnss_files.so -> ../../lib64/libnss_files.so.2 
-./usr/lib64/libnss_hesiod-2.29.so 
-./usr/lib64/libnss_hesiod.so.2 -> libnss_hesiod-2.29.so 
+./usr/lib64/libnss_hesiod-2.32.so 
+./usr/lib64/libnss_hesiod.so.2 -> libnss_hesiod-2.32.so 
 ./usr/lib64/libnss_hesiod.so -> ../../lib64/libnss_hesiod.so.2 
 ./usr/lib64/libnss_myhostname.so.2 
 ./usr/lib64/libnss_mymachines.so.2 
@@ -2397,8 +2397,8 @@
 ./usr/lib64/libprotoc.so.15.0.1 
 ./usr/lib64/libprotoc.so.15 -> libprotoc.so.15.0.1 
 ./usr/lib64/libprotoc.so -> libprotoc.so.15.0.1 
-./usr/lib64/libpthread-2.29.so 
-./usr/lib64/libpthread.so.0 -> libpthread-2.29.so 
+./usr/lib64/libpthread-2.32.so 
+./usr/lib64/libpthread.so.0 -> libpthread-2.32.so 
 ./usr/lib64/libpthread.so -> ../../lib64/libpthread.so.0 
 ./usr/lib64/libqpol.so.1.7 
 ./usr/lib64/libqpol.so.1 -> libqpol.so.1.7 
@@ -2412,14 +2412,14 @@
 ./usr/lib64/libref_array.so.1.2.0 
 ./usr/lib64/libref_array.so.1 -> libref_array.so.1.2.0 
 ./usr/lib64/libref_array.so -> libref_array.so.1.2.0 
-./usr/lib64/libresolv-2.29.so 
-./usr/lib64/libresolv.so.2 -> libresolv-2.29.so 
+./usr/lib64/libresolv-2.32.so 
+./usr/lib64/libresolv.so.2 -> libresolv-2.32.so 
 ./usr/lib64/libresolv.so -> ../../lib64/libresolv.so.2 
 ./usr/lib64/librootdev.so.1.0.0 
 ./usr/lib64/librootdev.so.1 -> librootdev.so.1.0.0 
 ./usr/lib64/librootdev.so -> librootdev.so.1.0.0 
-./usr/lib64/librt-2.29.so 
-./usr/lib64/librt.so.1 -> librt-2.29.so 
+./usr/lib64/librt-2.32.so 
+./usr/lib64/librt.so.1 -> librt-2.32.so 
 ./usr/lib64/librt.so -> ../../lib64/librt.so.1 
 ./usr/lib64/libsamba-credentials.so.0.0.1 
 ./usr/lib64/libsamba-credentials.so.0 -> libsamba-credentials.so.0.0.1 
@@ -2555,8 +2555,8 @@
 ./usr/lib64/libustr-1.0.so.1.0.4 
 ./usr/lib64/libustr-1.0.so.1 -> libustr-1.0.so.1.0.4 
 ./usr/lib64/libustr.so -> libustr-1.0.so.1 
-./usr/lib64/libutil-2.29.so 
-./usr/lib64/libutil.so.1 -> libutil-2.29.so 
+./usr/lib64/libutil-2.32.so 
+./usr/lib64/libutil.so.1 -> libutil-2.32.so 
 ./usr/lib64/libutil.so -> ../../lib64/libutil.so.1 
 ./usr/lib64/libuuid.so 
 ./usr/lib64/libuuid.so.1.3.0 
```
```

t-lo commented:
# Dev container diff (to Nov-25 nightly)
```
$ awk '{ s = ""; for (i = 8; i <= NF; i++) s = s $i " "; print s }' flatcar_developer_container_contents.txt | sort >dev-cont.txt
$ awk '{ s = ""; for (i = 8; i <= NF; i++) s = s $i " "; print s }' flatcar_developer_container_contents-glibc-2.32.txt | sort >dev-cont-glibc.txt
$ diff -ur prod-image-cont.txt prod-image-cont-glibc.txt
--- dev-cont.txt	2020-11-25 17:53:01.629522731 +0100
+++ dev-cont-glibc.txt	2020-11-25 17:53:17.163515611 +0100
@@ -1774,24 +1774,28 @@
 ./usr/include/awk/xalloc.h 
 ./usr/include/bits 
 ./usr/include/bits/a.out.h 
+./usr/include/bits/argp-ldbl.h 
 ./usr/include/bits/byteswap.h 
 ./usr/include/bits/cmathcalls.h 
 ./usr/include/bits/confname.h 
 ./usr/include/bits/cpu-set.h 
+./usr/include/bits/dirent_ext.h 
 ./usr/include/bits/dirent.h 
 ./usr/include/bits/dlfcn.h 
 ./usr/include/bits/elfclass.h 
 ./usr/include/bits/endian.h 
+./usr/include/bits/endianness.h 
 ./usr/include/bits/environments.h 
 ./usr/include/bits/epoll.h 
+./usr/include/bits/err-ldbl.h 
 ./usr/include/bits/errno.h 
 ./usr/include/bits/error.h 
+./usr/include/bits/error-ldbl.h 
 ./usr/include/bits/eventfd.h 
 ./usr/include/bits/fcntl2.h 
 ./usr/include/bits/fcntl.h 
 ./usr/include/bits/fcntl-linux.h 
 ./usr/include/bits/fenv.h 
-./usr/include/bits/fenvinline.h 
 ./usr/include/bits/floatn-common.h 
 ./usr/include/bits/floatn.h 
 ./usr/include/bits/flt-eval-method.h 
@@ -1808,6 +1812,7 @@
 ./usr/include/bits/ioctls.h 
 ./usr/include/bits/ioctl-types.h 
 ./usr/include/bits/ipc.h 
+./usr/include/bits/ipc-perm.h 
 ./usr/include/bits/ipctypes.h 
 ./usr/include/bits/iscanonical.h 
 ./usr/include/bits/libc-header-start.h 
@@ -1820,8 +1825,6 @@
 ./usr/include/bits/mathcalls-helper-functions.h 
 ./usr/include/bits/mathcalls-narrow.h 
 ./usr/include/bits/mathdef.h 
-./usr/include/bits/math-finite.h 
-./usr/include/bits/mathinline.h 
 ./usr/include/bits/math-vector.h 
 ./usr/include/bits/mman.h 
 ./usr/include/bits/mman-linux.h 
@@ -1831,7 +1834,6 @@
 ./usr/include/bits/mqueue2.h 
 ./usr/include/bits/mqueue.h 
 ./usr/include/bits/msq.h 
-./usr/include/bits/msq-pad.h 
 ./usr/include/bits/netdb.h 
 ./usr/include/bits/param.h 
 ./usr/include/bits/poll2.h 
@@ -1853,25 +1855,25 @@
 ./usr/include/bits/select.h 
 ./usr/include/bits/semaphore.h 
 ./usr/include/bits/sem.h 
-./usr/include/bits/sem-pad.h 
 ./usr/include/bits/setjmp2.h 
 ./usr/include/bits/setjmp.h 
 ./usr/include/bits/shm.h 
 ./usr/include/bits/shmlba.h 
-./usr/include/bits/shm-pad.h 
 ./usr/include/bits/sigaction.h 
 ./usr/include/bits/sigcontext.h 
 ./usr/include/bits/sigevent-consts.h 
 ./usr/include/bits/siginfo-arch.h 
 ./usr/include/bits/siginfo-consts-arch.h 
 ./usr/include/bits/siginfo-consts.h 
+./usr/include/bits/signal_ext.h 
 ./usr/include/bits/signalfd.h 
+./usr/include/bits/signum-arch.h 
 ./usr/include/bits/signum-generic.h 
-./usr/include/bits/signum.h 
 ./usr/include/bits/sigstack.h 
 ./usr/include/bits/sigthread.h 
 ./usr/include/bits/sockaddr.h 
 ./usr/include/bits/socket2.h 
+./usr/include/bits/socket-constants.h 
 ./usr/include/bits/socket.h 
 ./usr/include/bits/socket_type.h 
 ./usr/include/bits/ss_flags.h 
@@ -1879,6 +1881,7 @@
 ./usr/include/bits/statfs.h 
 ./usr/include/bits/stat.h 
 ./usr/include/bits/statvfs.h 
+./usr/include/bits/statx-generic.h 
 ./usr/include/bits/statx.h 
 ./usr/include/bits/stdint-intn.h 
 ./usr/include/bits/stdint-uintn.h 
@@ -1892,10 +1895,9 @@
 ./usr/include/bits/stdlib-ldbl.h 
 ./usr/include/bits/string_fortified.h 
 ./usr/include/bits/strings_fortified.h 
-./usr/include/bits/stropts.h 
+./usr/include/bits/struct_mutex.h 
+./usr/include/bits/struct_rwlock.h 
 ./usr/include/bits/syscall.h 
-./usr/include/bits/sysctl.h 
-./usr/include/bits/sys_errlist.h 
 ./usr/include/bits/syslog.h 
 ./usr/include/bits/syslog-ldbl.h 
 ./usr/include/bits/syslog-path.h 
@@ -1943,10 +1945,15 @@
 ./usr/include/bits/types/struct_FILE.h 
 ./usr/include/bits/types/struct_iovec.h 
 ./usr/include/bits/types/struct_itimerspec.h 
+./usr/include/bits/types/struct_msqid_ds.h 
 ./usr/include/bits/types/struct_osockaddr.h 
 ./usr/include/bits/types/struct_rusage.h 
 ./usr/include/bits/types/struct_sched_param.h 
+./usr/include/bits/types/struct_semid_ds.h 
+./usr/include/bits/types/struct_shmid_ds.h 
 ./usr/include/bits/types/struct_sigstack.h 
+./usr/include/bits/types/struct_statx.h 
+./usr/include/bits/types/struct_statx_timestamp.h 
 ./usr/include/bits/types/struct_timespec.h 
 ./usr/include/bits/types/struct_timeval.h 
 ./usr/include/bits/types/struct_tm.h 
@@ -1956,6 +1963,7 @@
 ./usr/include/bits/uintn-identity.h 
 ./usr/include/bits/uio-ext.h 
 ./usr/include/bits/uio_lim.h 
+./usr/include/bits/unistd_ext.h 
 ./usr/include/bits/unistd.h 
 ./usr/include/bits/utmp.h 
 ./usr/include/bits/utmpx.h 
@@ -1968,7 +1976,6 @@
 ./usr/include/bits/wctype-wchar.h 
 ./usr/include/bits/wordsize.h 
 ./usr/include/bits/xopen_lim.h 
-./usr/include/bits/xtitypes.h 
 ./usr/include/blkid 
 ./usr/include/blkid/blkid.h 
 ./usr/include/bsd 
@@ -2175,6 +2182,8 @@
 ./usr/include/fcntl.h 
 ./usr/include/features.h 
 ./usr/include/fenv.h 
+./usr/include/finclude 
+./usr/include/finclude/math-vector-fortran.h 
 ./usr/include/fmtmsg.h 
 ./usr/include/fnmatch.h 
 ./usr/include/form.h 
@@ -4963,7 +4972,6 @@
 ./usr/include/stdlib.h 
 ./usr/include/string.h 
 ./usr/include/strings.h 
-./usr/include/stropts.h 
 ./usr/include/sudo_plugin.h 
 ./usr/include/sys 
 ./usr/include/sys/acct.h 
@@ -5019,16 +5027,15 @@
 ./usr/include/sys/shm.h 
 ./usr/include/sys/signalfd.h 
 ./usr/include/sys/signal.h 
+./usr/include/sys/single_threaded.h 
 ./usr/include/sys/socket.h 
 ./usr/include/sys/socketvar.h 
 ./usr/include/sys/soundcard.h 
 ./usr/include/sys/statfs.h 
 ./usr/include/sys/stat.h 
 ./usr/include/sys/statvfs.h 
-./usr/include/sys/stropts.h 
 ./usr/include/sys/swap.h 
 ./usr/include/sys/syscall.h 
-./usr/include/sys/sysctl.h 
 ./usr/include/sys/sysinfo.h 
 ./usr/include/sys/syslog.h 
 ./usr/include/sys/sysmacros.h 
@@ -7573,8 +7580,8 @@
 ./usr/lib64/glib-2.0 
 ./usr/lib64/glib-2.0/include 
 ./usr/lib64/glib-2.0/include/glibconfig.h 
-./usr/lib64/glibc-2.29 
-./usr/lib64/glibc-2.29/libm-2.29.a 
+./usr/lib64/glibc-2.32 
+./usr/lib64/glibc-2.32/libm-2.32.a 
 ./usr/lib64/groff 
 ./usr/lib64/groff/groffer 
 ./usr/lib64/groff/groffer/func.pl 
@@ -7607,20 +7614,20 @@
 ./usr/lib64/krb5/plugins/preauth/test.so 
 ./usr/lib64/krb5/plugins/tls 
 ./usr/lib64/krb5/plugins/tls/k5tls.so 
-./usr/lib64/ld-2.29.so 
+./usr/lib64/ld-2.32.so 
 ./usr/lib64/ldb 
 ./usr/lib64/ldb/libldb-cmdline.so 
 ./usr/lib64/ldb/libreplace-ldb.so 
-./usr/lib64/ld-linux-x86-64.so.2 -> ld-2.29.so 
+./usr/lib64/ld-linux-x86-64.so.2 -> ld-2.32.so 
 ./usr/lib64/libacl.so 
 ./usr/lib64/libacl.so.1.1.0 
 ./usr/lib64/libacl.so.1 -> libacl.so.1.1.0 
 ./usr/lib64/libaio.so 
 ./usr/lib64/libaio.so.1.0.1 
 ./usr/lib64/libaio.so.1 -> libaio.so.1.0.1 
-./usr/lib64/libanl-2.29.so 
+./usr/lib64/libanl-2.32.so 
 ./usr/lib64/libanl.a 
-./usr/lib64/libanl.so.1 -> libanl-2.29.so 
+./usr/lib64/libanl.so.1 -> libanl-2.32.so 
 ./usr/lib64/libanl.so -> ../../lib64/libanl.so.1 
 ./usr/lib64/libapol.a 
 ./usr/lib64/libapol.so.4.4 
@@ -7655,9 +7662,9 @@
 ./usr/lib64/libblkid.so 
 ./usr/lib64/libblkid.so.1.1.0 
 ./usr/lib64/libblkid.so.1 -> libblkid.so.1.1.0 
-./usr/lib64/libBrokenLocale-2.29.so 
+./usr/lib64/libBrokenLocale-2.32.so 
 ./usr/lib64/libBrokenLocale.a 
-./usr/lib64/libBrokenLocale.so.1 -> libBrokenLocale-2.29.so 
+./usr/lib64/libBrokenLocale.so.1 -> libBrokenLocale-2.32.so 
 ./usr/lib64/libBrokenLocale.so -> ../../lib64/libBrokenLocale.so.1 
 ./usr/lib64/libbsd-ctor.a 
 ./usr/lib64/libbsd.so.0.10.0 
@@ -7673,7 +7680,7 @@
 ./usr/lib64/libbz2.so.1.0.8 
 ./usr/lib64/libbz2.so.1.0 -> libbz2.so.1.0.8 
 ./usr/lib64/libbz2.so.1 -> libbz2.so.1.0.8 
-./usr/lib64/libc-2.29.so 
+./usr/lib64/libc-2.32.so 
 ./usr/lib64/libc.a 
 ./usr/lib64/libcap-ng.so.0.0.0 
 ./usr/lib64/libcap-ng.so.0 -> libcap-ng.so.0.0.0 
@@ -7694,17 +7701,17 @@
 ./usr/lib64/libcom_err.so 
 ./usr/lib64/libcom_err.so.2.1 
 ./usr/lib64/libcom_err.so.2 -> libcom_err.so.2.1 
-./usr/lib64/libcrypt-2.29.so 
+./usr/lib64/libcrypt-2.32.so 
 ./usr/lib64/libcrypt.a 
 ./usr/lib64/libcrypto.so.1.1 
 ./usr/lib64/libcrypto.so -> libcrypto.so.1.1 
 ./usr/lib64/libcryptsetup.so.12.6.0 
 ./usr/lib64/libcryptsetup.so.12 -> libcryptsetup.so.12.6.0 
 ./usr/lib64/libcryptsetup.so -> libcryptsetup.so.12.6.0 
-./usr/lib64/libcrypt.so.1 -> libcrypt-2.29.so 
+./usr/lib64/libcrypt.so.1 -> libcrypt-2.32.so 
 ./usr/lib64/libcrypt.so -> ../../lib64/libcrypt.so.1 
 ./usr/lib64/libc.so 
-./usr/lib64/libc.so.6 -> libc-2.29.so 
+./usr/lib64/libc.so.6 -> libc-2.32.so 
 ./usr/lib64/libcurl.so.4.6.0 
 ./usr/lib64/libcurl.so.4 -> libcurl.so.4.6.0 
 ./usr/lib64/libcurl.so -> libcurl.so.4.6.0 
@@ -7737,9 +7744,9 @@
 ./usr/lib64/libdhash.so.1.0.2 
 ./usr/lib64/libdhash.so.1 -> libdhash.so.1.0.2 
 ./usr/lib64/libdhash.so -> libdhash.so.1.0.2 
-./usr/lib64/libdl-2.29.so 
+./usr/lib64/libdl-2.32.so 
 ./usr/lib64/libdl.a 
-./usr/lib64/libdl.so.2 -> libdl-2.29.so 
+./usr/lib64/libdl.so.2 -> libdl-2.32.so 
 ./usr/lib64/libdl.so -> ../../lib64/libdl.so.2 
 ./usr/lib64/libdw-0.178.so 
 ./usr/lib64/libdw.so.1 -> libdw-0.178.so 
@@ -7974,7 +7981,7 @@
 ./usr/lib64/liblzo2.so.2.0.0 
 ./usr/lib64/liblzo2.so.2 -> liblzo2.so.2.0.0 
 ./usr/lib64/liblzo2.so -> liblzo2.so.2.0.0 
-./usr/lib64/libm-2.29.so 
+./usr/lib64/libm-2.32.so 
 ./usr/lib64/libm.a 
 ./usr/lib64/libmagic.so.1.0.0 
 ./usr/lib64/libmagic.so.1 -> libmagic.so.1.0.0 
@@ -8010,13 +8017,12 @@
 ./usr/lib64/libmpfr.so.4 -> libmpfr.so.4.1.6 
 ./usr/lib64/libmpfr.so -> libmpfr.so.4.1.6 
 ./usr/lib64/libm.so 
-./usr/lib64/libm.so.6 -> libm-2.29.so 
+./usr/lib64/libm.so.6 -> libm-2.32.so 
 ./usr/lib64/libmultipath.so.0 
 ./usr/lib64/libmultipath.so -> libmultipath.so.0 
-./usr/lib64/libmvec-2.29.so 
+./usr/lib64/libmvec-2.32.so 
 ./usr/lib64/libmvec.a 
-./usr/lib64/libmvec_nonshared.a 
-./usr/lib64/libmvec.so.1 -> libmvec-2.29.so 
+./usr/lib64/libmvec.so.1 -> libmvec-2.32.so 
 ./usr/lib64/libmvec.so -> ../../lib64/libmvec.so.1 
 ./usr/lib64/libncurses.so 
 ./usr/lib64/libncurses.so.6.1 
@@ -8090,25 +8096,25 @@
 ./usr/lib64/libnpth.so.0.0.6 
 ./usr/lib64/libnpth.so.0 -> libnpth.so.0.0.6 
 ./usr/lib64/libnpth.so -> libnpth.so.0.0.6 
-./usr/lib64/libnsl-2.29.so 
-./usr/lib64/libnsl.so.1 -> libnsl-2.29.so 
+./usr/lib64/libnsl-2.32.so 
+./usr/lib64/libnsl.so.1 -> libnsl-2.32.so 
 ./usr/lib64/libnspr4.so 
 ./usr/lib64/libnss_cache_oslogin-20200910.00.so 
 ./usr/lib64/libnss_cache_oslogin.so.2 -> libnss_cache_oslogin-20200910.00.so 
-./usr/lib64/libnss_compat-2.29.so 
-./usr/lib64/libnss_compat.so.2 -> libnss_compat-2.29.so 
+./usr/lib64/libnss_compat-2.32.so 
+./usr/lib64/libnss_compat.so.2 -> libnss_compat-2.32.so 
 ./usr/lib64/libnss_compat.so -> ../../lib64/libnss_compat.so.2 
-./usr/lib64/libnss_db-2.29.so 
-./usr/lib64/libnss_db.so.2 -> libnss_db-2.29.so 
+./usr/lib64/libnss_db-2.32.so 
+./usr/lib64/libnss_db.so.2 -> libnss_db-2.32.so 
 ./usr/lib64/libnss_db.so -> ../../lib64/libnss_db.so.2 
-./usr/lib64/libnss_dns-2.29.so 
-./usr/lib64/libnss_dns.so.2 -> libnss_dns-2.29.so 
+./usr/lib64/libnss_dns-2.32.so 
+./usr/lib64/libnss_dns.so.2 -> libnss_dns-2.32.so 
 ./usr/lib64/libnss_dns.so -> ../../lib64/libnss_dns.so.2 
-./usr/lib64/libnss_files-2.29.so 
-./usr/lib64/libnss_files.so.2 -> libnss_files-2.29.so 
+./usr/lib64/libnss_files-2.32.so 
+./usr/lib64/libnss_files.so.2 -> libnss_files-2.32.so 
 ./usr/lib64/libnss_files.so -> ../../lib64/libnss_files.so.2 
-./usr/lib64/libnss_hesiod-2.29.so 
-./usr/lib64/libnss_hesiod.so.2 -> libnss_hesiod-2.29.so 
+./usr/lib64/libnss_hesiod-2.32.so 
+./usr/lib64/libnss_hesiod.so.2 -> libnss_hesiod-2.32.so 
 ./usr/lib64/libnss_hesiod.so -> ../../lib64/libnss_hesiod.so.2 
 ./usr/lib64/libnss_myhostname.so.2 
 ./usr/lib64/libnss_mymachines.so.2 
@@ -8210,9 +8216,9 @@
 ./usr/lib64/libprotoc.so.15.0.1 
 ./usr/lib64/libprotoc.so.15 -> libprotoc.so.15.0.1 
 ./usr/lib64/libprotoc.so -> libprotoc.so.15.0.1 
-./usr/lib64/libpthread-2.29.so 
+./usr/lib64/libpthread-2.32.so 
 ./usr/lib64/libpthread.a 
-./usr/lib64/libpthread.so.0 -> libpthread-2.29.so 
+./usr/lib64/libpthread.so.0 -> libpthread-2.32.so 
 ./usr/lib64/libpthread.so -> ../../lib64/libpthread.so.0 
 ./usr/lib64/libpython2.7.a 
 ./usr/lib64/libpython2.7.so.1.0 
@@ -8230,16 +8236,16 @@
 ./usr/lib64/libref_array.so.1.2.0 
 ./usr/lib64/libref_array.so.1 -> libref_array.so.1.2.0 
 ./usr/lib64/libref_array.so -> libref_array.so.1.2.0 
-./usr/lib64/libresolv-2.29.so 
+./usr/lib64/libresolv-2.32.so 
 ./usr/lib64/libresolv.a 
-./usr/lib64/libresolv.so.2 -> libresolv-2.29.so 
+./usr/lib64/libresolv.so.2 -> libresolv-2.32.so 
 ./usr/lib64/libresolv.so -> ../../lib64/libresolv.so.2 
 ./usr/lib64/librootdev.so.1.0.0 
 ./usr/lib64/librootdev.so.1 -> librootdev.so.1.0.0 
 ./usr/lib64/librootdev.so -> librootdev.so.1.0.0 
-./usr/lib64/librt-2.29.so 
+./usr/lib64/librt-2.32.so 
 ./usr/lib64/librt.a 
-./usr/lib64/librt.so.1 -> librt-2.29.so 
+./usr/lib64/librt.so.1 -> librt-2.32.so 
 ./usr/lib64/librt.so -> ../../lib64/librt.so.1 
 ./usr/lib64/libsamba-credentials.so.0.0.1 
 ./usr/lib64/libsamba-credentials.so.0 -> libsamba-credentials.so.0.0.1 
@@ -8372,9 +8378,9 @@
 ./usr/lib64/libustr-1.0.so.1.0.4 
 ./usr/lib64/libustr-1.0.so.1 -> libustr-1.0.so.1.0.4 
 ./usr/lib64/libustr.so -> libustr-1.0.so.1 
-./usr/lib64/libutil-2.29.so 
+./usr/lib64/libutil-2.32.so 
 ./usr/lib64/libutil.a 
-./usr/lib64/libutil.so.1 -> libutil-2.29.so 
+./usr/lib64/libutil.so.1 -> libutil-2.32.so 
 ./usr/lib64/libutil.so -> ../../lib64/libutil.so.1 
 ./usr/lib64/libuuid.so 
 ./usr/lib64/libuuid.so.1.3.0 
@@ -39581,6 +39587,7 @@
 ./usr/share/i18n/locales/ca_IT 
 ./usr/share/i18n/locales/ce_RU 
 ./usr/share/i18n/locales/chr_US 
+./usr/share/i18n/locales/ckb_IQ 
 ./usr/share/i18n/locales/cmn_TW 
 ./usr/share/i18n/locales/cns11643_stroke 
 ./usr/share/i18n/locales/crh_UA 
@@ -39743,6 +39750,7 @@
 ./usr/share/i18n/locales/ml_IN 
 ./usr/share/i18n/locales/mni_IN 
 ./usr/share/i18n/locales/mn_MN 
+./usr/share/i18n/locales/mnw_MM 
 ./usr/share/i18n/locales/mr_IN 
 ./usr/share/i18n/locales/ms_MY 
 ./usr/share/i18n/locales/mt_MT 
@@ -43796,11 +43804,11 @@
 ./var/cache/edb/binhost/flatcar-jenkins/developer 
 ./var/cache/edb/binhost/flatcar-jenkins/developer/boards 
 ./var/cache/edb/binhost/flatcar-jenkins/developer/boards/amd64-usr 
-./var/cache/edb/binhost/flatcar-jenkins/developer/boards/amd64-usr/2020.11.25+dev-main-nightly-1541 
-./var/cache/edb/binhost/flatcar-jenkins/developer/boards/amd64-usr/2020.11.25+dev-main-nightly-1541/pkgs 
-./var/cache/edb/binhost/flatcar-jenkins/developer/boards/amd64-usr/2020.11.25+dev-main-nightly-1541/pkgs/Packages 
-./var/cache/edb/binhost/flatcar-jenkins/developer/boards/amd64-usr/2020.11.25+dev-main-nightly-1541/toolchain 
-./var/cache/edb/binhost/flatcar-jenkins/developer/boards/amd64-usr/2020.11.25+dev-main-nightly-1541/toolchain/Packages 
+./var/cache/edb/binhost/flatcar-jenkins/developer/boards/amd64-usr/2020.11.24+dev-flatcar-master-1540 
+./var/cache/edb/binhost/flatcar-jenkins/developer/boards/amd64-usr/2020.11.24+dev-flatcar-master-1540/pkgs 
+./var/cache/edb/binhost/flatcar-jenkins/developer/boards/amd64-usr/2020.11.24+dev-flatcar-master-1540/pkgs/Packages 
+./var/cache/edb/binhost/flatcar-jenkins/developer/boards/amd64-usr/2020.11.24+dev-flatcar-master-1540/toolchain 
+./var/cache/edb/binhost/flatcar-jenkins/developer/boards/amd64-usr/2020.11.24+dev-flatcar-master-1540/toolchain/Packages 
 ./var/cache/edb/counter 
 ./var/cache/edb/mtimedb 
 ./var/cache/edb/vdb_metadata_delta.json 
@@ -48429,7 +48437,6 @@
 ./var/db/pkg/net-dns/dnsmasq-2.82/SLOT 
 ./var/db/pkg/net-dns/dnsmasq-2.82/USE 
 ./var/db/pkg/net-dns/libidn2-2.3.0 
-./var/db/pkg/net-dns/libidn2-2.3.0/BDEPEND 
 ./var/db/pkg/net-dns/libidn2-2.3.0/BINPKGMD5 
 ./var/db/pkg/net-dns/libidn2-2.3.0/BUILD_TIME 
 ./var/db/pkg/net-dns/libidn2-2.3.0/CATEGORY 
@@ -52511,46 +52518,45 @@
 ./var/db/pkg/sys-libs/gdbm-1.11/SIZE 
 ./var/db/pkg/sys-libs/gdbm-1.11/SLOT 
 ./var/db/pkg/sys-libs/gdbm-1.11/USE 
-./var/db/pkg/sys-libs/glibc-2.29-r2 
-./var/db/pkg/sys-libs/glibc-2.29-r2/BINPKGMD5 
-./var/db/pkg/sys-libs/glibc-2.29-r2/BUILD_TIME 
-./var/db/pkg/sys-libs/glibc-2.29-r2/CATEGORY 
-./var/db/pkg/sys-libs/glibc-2.29-r2/CBUILD 
-./var/db/pkg/sys-libs/glibc-2.29-r2/CC 
-./var/db/pkg/sys-libs/glibc-2.29-r2/CFLAGS 
-./var/db/pkg/sys-libs/glibc-2.29-r2/CHOST 
-./var/db/pkg/sys-libs/glibc-2.29-r2/CONTENTS 
-./var/db/pkg/sys-libs/glibc-2.29-r2/COUNTER 
-./var/db/pkg/sys-libs/glibc-2.29-r2/CTARGET 
-./var/db/pkg/sys-libs/glibc-2.29-r2/CXX 
-./var/db/pkg/sys-libs/glibc-2.29-r2/CXXFLAGS 
-./var/db/pkg/sys-libs/glibc-2.29-r2/DEBUGBUILD 
-./var/db/pkg/sys-libs/glibc-2.29-r2/DEFINED_PHASES 
-./var/db/pkg/sys-libs/glibc-2.29-r2/DEPEND 
-./var/db/pkg/sys-libs/glibc-2.29-r2/DESCRIPTION 
-./var/db/pkg/sys-libs/glibc-2.29-r2/EAPI 
-./var/db/pkg/sys-libs/glibc-2.29-r2/environment.bz2 
-./var/db/pkg/sys-libs/glibc-2.29-r2/FEATURES 
-./var/db/pkg/sys-libs/glibc-2.29-r2/glibc-2.29-r2.ebuild 
-./var/db/pkg/sys-libs/glibc-2.29-r2/HOMEPAGE 
-./var/db/pkg/sys-libs/glibc-2.29-r2/INHERITED 
-./var/db/pkg/sys-libs/glibc-2.29-r2/INSTALL_MASK 
-./var/db/pkg/sys-libs/glibc-2.29-r2/IUSE 
-./var/db/pkg/sys-libs/glibc-2.29-r2/IUSE_EFFECTIVE 
-./var/db/pkg/sys-libs/glibc-2.29-r2/KEYWORDS 
-./var/db/pkg/sys-libs/glibc-2.29-r2/LDFLAGS 
-./var/db/pkg/sys-libs/glibc-2.29-r2/LICENSE 
-./var/db/pkg/sys-libs/glibc-2.29-r2/NEEDED 
-./var/db/pkg/sys-libs/glibc-2.29-r2/NEEDED.ELF.2 
-./var/db/pkg/sys-libs/glibc-2.29-r2/PDEPEND 
-./var/db/pkg/sys-libs/glibc-2.29-r2/PF 
-./var/db/pkg/sys-libs/glibc-2.29-r2/PROVIDES 
-./var/db/pkg/sys-libs/glibc-2.29-r2/RDEPEND 
-./var/db/pkg/sys-libs/glibc-2.29-r2/repository 
-./var/db/pkg/sys-libs/glibc-2.29-r2/RESTRICT 
-./var/db/pkg/sys-libs/glibc-2.29-r2/SIZE 
-./var/db/pkg/sys-libs/glibc-2.29-r2/SLOT 
-./var/db/pkg/sys-libs/glibc-2.29-r2/USE 
+./var/db/pkg/sys-libs/glibc-2.32-r2 
+./var/db/pkg/sys-libs/glibc-2.32-r2/BINPKGMD5 
+./var/db/pkg/sys-libs/glibc-2.32-r2/BUILD_TIME 
+./var/db/pkg/sys-libs/glibc-2.32-r2/CATEGORY 
+./var/db/pkg/sys-libs/glibc-2.32-r2/CBUILD 
+./var/db/pkg/sys-libs/glibc-2.32-r2/CC 
+./var/db/pkg/sys-libs/glibc-2.32-r2/CFLAGS 
+./var/db/pkg/sys-libs/glibc-2.32-r2/CHOST 
+./var/db/pkg/sys-libs/glibc-2.32-r2/CONTENTS 
+./var/db/pkg/sys-libs/glibc-2.32-r2/COUNTER 
+./var/db/pkg/sys-libs/glibc-2.32-r2/CTARGET 
+./var/db/pkg/sys-libs/glibc-2.32-r2/CXX 
+./var/db/pkg/sys-libs/glibc-2.32-r2/CXXFLAGS 
+./var/db/pkg/sys-libs/glibc-2.32-r2/DEFINED_PHASES 
+./var/db/pkg/sys-libs/glibc-2.32-r2/DEPEND 
+./var/db/pkg/sys-libs/glibc-2.32-r2/DESCRIPTION 
+./var/db/pkg/sys-libs/glibc-2.32-r2/EAPI 
+./var/db/pkg/sys-libs/glibc-2.32-r2/environment.bz2 
+./var/db/pkg/sys-libs/glibc-2.32-r2/FEATURES 
+./var/db/pkg/sys-libs/glibc-2.32-r2/glibc-2.32-r2.ebuild 
+./var/db/pkg/sys-libs/glibc-2.32-r2/HOMEPAGE 
+./var/db/pkg/sys-libs/glibc-2.32-r2/INHERITED 
+./var/db/pkg/sys-libs/glibc-2.32-r2/INSTALL_MASK 
+./var/db/pkg/sys-libs/glibc-2.32-r2/IUSE 
+./var/db/pkg/sys-libs/glibc-2.32-r2/IUSE_EFFECTIVE 
+./var/db/pkg/sys-libs/glibc-2.32-r2/KEYWORDS 
+./var/db/pkg/sys-libs/glibc-2.32-r2/LDFLAGS 
+./var/db/pkg/sys-libs/glibc-2.32-r2/LICENSE 
+./var/db/pkg/sys-libs/glibc-2.32-r2/NEEDED 
+./var/db/pkg/sys-libs/glibc-2.32-r2/NEEDED.ELF.2 
+./var/db/pkg/sys-libs/glibc-2.32-r2/PDEPEND 
+./var/db/pkg/sys-libs/glibc-2.32-r2/PF 
+./var/db/pkg/sys-libs/glibc-2.32-r2/PROVIDES 
+./var/db/pkg/sys-libs/glibc-2.32-r2/RDEPEND 
+./var/db/pkg/sys-libs/glibc-2.32-r2/repository 
+./var/db/pkg/sys-libs/glibc-2.32-r2/RESTRICT 
+./var/db/pkg/sys-libs/glibc-2.32-r2/SIZE 
+./var/db/pkg/sys-libs/glibc-2.32-r2/SLOT 
+./var/db/pkg/sys-libs/glibc-2.32-r2/USE 
 ./var/db/pkg/sys-libs/ldb-1.3.6 
 ./var/db/pkg/sys-libs/ldb-1.3.6/BINPKGMD5 
 ./var/db/pkg/sys-libs/ldb-1.3.6/BUILD_TIME 
@@ -53262,6 +53268,33 @@
 ./var/db/pkg/sys-process/psmisc-22.21-r2/SLOT 
 ./var/db/pkg/sys-process/psmisc-22.21-r2/USE 
 ./var/db/pkg/virtual 
+./var/db/pkg/virtual/awk-1 
+./var/db/pkg/virtual/awk-1/awk-1.ebuild 
+./var/db/pkg/virtual/awk-1/BINPKGMD5 
+./var/db/pkg/virtual/awk-1/BUILD_TIME 
+./var/db/pkg/virtual/awk-1/CATEGORY 
+./var/db/pkg/virtual/awk-1/CBUILD 
+./var/db/pkg/virtual/awk-1/CFLAGS 
+./var/db/pkg/virtual/awk-1/CHOST 
+./var/db/pkg/virtual/awk-1/CONTENTS 
+./var/db/pkg/virtual/awk-1/COUNTER 
+./var/db/pkg/virtual/awk-1/CXXFLAGS 
+./var/db/pkg/virtual/awk-1/DEFINED_PHASES 
+./var/db/pkg/virtual/awk-1/DESCRIPTION 
+./var/db/pkg/virtual/awk-1/EAPI 
+./var/db/pkg/virtual/awk-1/environment.bz2 
+./var/db/pkg/virtual/awk-1/FEATURES 
+./var/db/pkg/virtual/awk-1/INSTALL_MASK 
+./var/db/pkg/virtual/awk-1/IUSE 
+./var/db/pkg/virtual/awk-1/IUSE_EFFECTIVE 
+./var/db/pkg/virtual/awk-1/KEYWORDS 
+./var/db/pkg/virtual/awk-1/LDFLAGS 
+./var/db/pkg/virtual/awk-1/PF 
+./var/db/pkg/virtual/awk-1/RDEPEND 
+./var/db/pkg/virtual/awk-1/repository 
+./var/db/pkg/virtual/awk-1/SIZE 
+./var/db/pkg/virtual/awk-1/SLOT 
+./var/db/pkg/virtual/awk-1/USE 
 ./var/db/pkg/virtual/dev-manager-0-r1 
 ./var/db/pkg/virtual/dev-manager-0-r1/BINPKGMD5 
 ./var/db/pkg/virtual/dev-manager-0-r1/BUILD_TIME 
```
